static/app/views/settings/project/projectOwnership/ruleBuilder.tsx

```diff
@@ -228,7 +228,6 @@ const StyledIconAdd = styled(IconAdd)`
 
 const BuilderBar = styled('div')`
   display: flex;
-  height: 40px;
   align-items: center;
   margin-bottom: ${space(2)};
 `;
```

Comment:
`BuilderBar` doesn't need an explicit height, it'll just expand to accommodate its children.
<img width="601" alt="srcexample" src="https://user-images.githubusercontent.com/44172267/196279412-0fd2d465-6c13-46dd-9d6a-8d5ed5ba2898.png">